### PR TITLE
Fix issue DPTP-3609 Rename origin-ci-test to test-platform-results

### PR DESCRIPTION
**Pull request by bear-redhat**
1. pj-reherase config changed (add a new parameter)
2. batch replacement in existing files

/hold

### CODE REVIEW

File: cluster/test-deploy/gcp-crio/vars-origin.yaml [hunk 1]
```diff
@@ -44,7 +44,7 @@ openshift_schedulable: True
 openshift_hosted_registry_storage_gcs_keyfile: "{{ files_dir }}/gce.json"
 openshift_hosted_registry_storage_kind: object
 openshift_hosted_registry_storage_provider: gcs
-openshift_hosted_registry_storage_gcs_bucket: origin-ci-test-{{ lookup('env', 'INSTANCE_PREFIX') | mandatory }}
```
Comment (jupierce) [inline cluster/test-deploy/gcp-crio/vars-origin.yaml:47]: There is only one `test-platform-results` bucket. `INSTANCE_PREFIX` appears to be allow access to other buckets that start with `origin-ci-test-` . Those are private and we don't need to migrate them.

File: clusters/app.ci/app.ci/prow/staging/gcsweb.yaml [hunk 2]
```diff
@@ -24,7 +24,7 @@ items:
           name: gcsweb
           args:
           - "-b"
-          - "origin-ci-test"
```
Comment (jupierce) [inline clusters/app.ci/app.ci/prow/staging/gcsweb.yaml:27]: Use https://github.com/kubernetes/test-infra/pull/31456 and deploy with alias to `origin-ci-test` .

File: clusters/app.ci/prow/03_deployment/pj-rehearse-plugin.yaml [hunk 3]
```diff
@@ -65,6 +65,7 @@ items:
           - --github-endpoint=https://api.github.com
           - --github-token-path=/etc/github/oauth
           - --config-path=/etc/config/config.yaml
+          - --gsc-bucket=test-platform-results
```
Comment (jupierce) [inline clusters/app.ci/prow/03_deployment/pj-rehearse-plugin.yaml:68]: ?
```suggestion
          - --gcs-bucket=test-platform-results
```

Comment (jupierce) [inline clusters/app.ci/prow/03_deployment/pj-rehearse-plugin.yaml:68]: There's also `gcs-browser-prefix` .

**Review by jupierce — CHANGES_REQUESTED** [review]
What about https://github.com/openshift/ci-tools ?

File: clusters/app.ci/prow/03_deployment/pj-rehearse-plugin.yaml (hunk 3 above)
Comment (bear-redhat) [reply clusters/app.ci/prow/03_deployment/pj-rehearse-plugin.yaml:68]: I think this one will be covered by the alias right?

**Comment by bear-redhat** [comment]
> What about https://github.com/openshift/ci-tools ?

Ref: https://redhat-internal.slack.com/archives/C06AFJN03NC/p1702669073425639

**Pull request closed without merging**